omnibus/config/software/datadog-agent-finalize.rb

```diff
@@ -95,8 +95,12 @@
             command "find #{install_dir}/embedded '(' -name '*.pyc' -o -name '*.pyo' ')' -type f -delete -print >> #{install_dir}/embedded/.py_compiled_files.txt"
 
             # Setup pip aliases: `/opt/datadog-agent/embedded/bin/pip` will default to `pip2`
-            delete "#{install_dir}/embedded/bin/pip"
-            link "#{install_dir}/embedded/bin/pip2", "#{install_dir}/embedded/bin/pip"
+            python_runtimes = ENV['PYTHON_RUNTIMES'].split(',')
+            if python_runtimes.include? "2"
+                delete "#{install_dir}/embedded/bin/pip"
+                link "#{install_dir}/embedded/bin/pip2", "#{install_dir}/embedded/bin/pip"
+            end
```

Comment:
we can probably add an `else` and link `pip` to `pip3` as a fall back

Reply:
If only python3 is installed we know for sure that `pip` already points to `pip3`.